projects/gnomad/src/server/server.js

```diff
@@ -29,8 +38,56 @@ if (JSON.parse(process.env.ENABLE_HTTPS_REDIRECT || 'false')) {
   })
 }
 
+//var myLogger = function (req, res, next) {
+
+//  var wholeCookie = req.cookies
+//  var chorus_accessToken = req.cookies['accessToken']
+//  console.log(chorus_accessToken)
+
+//  var jwtDecode = require('jwt-decode')
+//  let decoded = jwtDecode(chorus_accessToken)
+//  var current_time = (new Date().getTime() / 1000)
+  
+//  if (current_time > decoded.exp){
+//    console.log('EXPIRED')
+//    res.redirect(`127.0.0.1/auth`)  
+//  }
+//  next()
+//}
+//app.use(myLogger)
+
+app.use((request, response, next) => {
+  // get cookies
+  var isRedirect = true
+  var jwtDecode = require('jwt-decode')
```

Comment:
```suggestion
  import jwtDecode from 'jwt-decode'
```
Prefer an import syntax on top of the code